src/IO/SensorLogger.cpp

```diff
@@ -174,8 +207,72 @@ void SensorLogger::writeHeader(std::ofstream &fileStream)
 
     fileStream << "solutionStatus,";
 
-    fileStream << "currentStateNo,\n";
+    fileStream << "gpsPosStatus,";
 
+    fileStream << "gpsPosAccurracyLatitude,";
+    fileStream << "gpsPosAccuracyLongitude,";
+    fileStream << "gpsPosAccuracyAltitude,";
+
+    fileStream << "NumSvUsed,";
+
+    fileStream << "velocityNAccuracy,";
+    fileStream << "velocityEAccuracy,";
+    fileStream << "velocityDAccuracy,";
+
+    fileStream << "latitudeAccuracy,";
+    fileStream << "longitudeAccuracy,";
+    fileStream << "altitudeAccuracy,";
+
+    fileStream << "pressureStatus,";
+    fileStream << "barometricPressure,";
+
+    fileStream << "imuStatus,";
+
+    fileStream << "gyroX,";
+    fileStream << "gyroY,";
+    fileStream << "gyroZ,";
+
+    fileStream << "temp,";
+
+    fileStream << "deltaVelX,";
+    fileStream << "deltaVelY,";
+    fileStream << "deltaVelZ,";
+
+    fileStream << "deltaAngleX,";
+    fileStream << "deltaAngleY,";
+    fileStream << "deltaAngleZ,";
+#endif
+
+// Initialization data
+#if USE_LOGGER
+    fileStream << "loggerInitialized, ";
+#endif
+
+#if USE_SOCKET_CLIENT
+    fileStream << "clientInitialized, ";
+    fileStream << "lastActiveClientTimestamp, ";
+#endif
+
+#if USE_SBG
+    fileStream << "sbgInitialized, ";
+#endif
+
+#if USE_INPUT
+    fileStream << "inputInitialized, ";
+#endif
+
+#if USE_RADIO
+    fileStream << "radioInitialized, ";
+#endif
+
+#if USE_GPIO
+    fileStream << "gpioInitialized, ";
+#endif
+
+#if USE_ARDUINO_PROXY
+    fileStream << "arduinoProxyInitialized, ";
```

Comment:
Some of these have extra spaces at the end, this is causing the test not to work, as there should be a blank space at the end of the line due to

```
fileStream << "sbgInitialized, ";
```